spec/features/welcome/index_spec.rb

```diff
@@ -16,8 +15,7 @@
       within ('.description') do
         expect(page).to have_content('This app allows you to glitch a photo that is created by an AI based on a word or phrase that you provide.') # Just a placeholder - Need an actual description
       end
-      expect(page).to have_css('img[src="https://www.webdesignbites.com/wp-content/uploads/2019/04/glitch-art.jpg"]') #"img[src=\"#{link}\"]"
-      expect(page).to have_button('Login with Google to Start Glitching')
+      expect(page).to have_link('Login with Google to Start Glitching')
```

Comment:
Turns out Bootstrap makes this a link and not button, even though it looks like a button.